### DISCUSSION TITLE
Dropdown menu scroll posotion resets to top when opened

### DISCUSSION THREAD

**Discussion by lewisblackburn** [epicweb-dev/epic-stack#346] [Q&A] [answered]
The scroll position of the page is unexpectedly reset to the top when opening a dropdown menu. I have recreated the problem with remix + shadcn-ui and the scroll position doesn't reset. This is also prevalent in the user dropdown. I'm not sure if this is intended behaviour so apologies if this is the case!

[Screencast from 23-07-23 19:53:54.webm](https://github.com/epicweb-dev/epic-stack/assets/51877955/f044e4be-16bf-4960-bbb1-66123b6de235)

**Comment by kentcdodds** [comment, marked as answer]
Without seeing the code it's hard to say what could be causing this. Is that a Link you're clicking on? If so, you may consider using: [`preventScrollReset`](https://reactrouter.com/en/main/components/link#preventscrollreset)
